src/components/History/HistoryList.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import HistoryItem from './HistoryItem';
+
+const HistoryList = ({ history }) => (
+  <ul>
+    {history.map(item => (
```

Comment:
I find it a bit more readable if you do the mapping outside the jsx and pass the elements in